src/fastga/models/weight/mass_breakdown/a_airframe/a2_fuselage_weight.py

```diff
@@ -78,6 +81,121 @@ def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
             a2 * inputs["data:weight:airframe:fuselage:k_factor"]
         )
 
+    # pylint: disable=missing-function-docstring, unused-argument
+    # Overriding OpenMDAO compute_partials, not all arguments are used
+    def compute_partials(self, inputs, partials, discrete_inputs=None):
+        sizing_factor_ultimate = inputs["data:mission:sizing:cs23:sizing_factor:ultimate_aircraft"]
+        mtow = inputs["data:weight:aircraft:MTOW"]
+        maximum_width = inputs["data:geometry:fuselage:maximum_width"]
+        maximum_height = inputs["data:geometry:fuselage:maximum_height"]
+        fus_length = inputs["data:geometry:fuselage:length"]
+        v_max_sl = inputs["data:TLAR:v_max_sl"]
+        k_factor = inputs["data:weight:airframe:fuselage:k_factor"]
+
+        partials[
+            "data:weight:airframe:fuselage:mass",
+            "data:mission:sizing:cs23:sizing_factor:ultimate_aircraft",
+        ] = k_factor * (
+            200.0
+            * 0.3146
+            * (
+                (mtow / (10.0**5.0)) ** 0.286
+                * (fus_length * 3.28084 / 10.0) ** 0.857
+                * (maximum_width + maximum_height)
+                * 3.28084
+                / 10.0
+                * (v_max_sl / 100.0) ** 0.338
+            )
+            ** 1.1
+            * sizing_factor_ultimate**-0.6854
+        )
+        partials["data:weight:airframe:fuselage:mass", "data:weight:aircraft:MTOW"] = k_factor * (
+            200.0
+            * 0.3146
+            * (
+                (sizing_factor_ultimate / (10.0**5.0)) ** 0.286
+                * (fus_length * 3.28084 / 10.0) ** 0.857
+                * (maximum_width + maximum_height)
+                * 3.28084
+                / 10.0
+                * (v_max_sl / 100.0) ** 0.338
+            )
+            ** 1.1
+            * mtow**-0.6854
+        )
+        partials["data:weight:airframe:fuselage:mass", "data:geometry:fuselage:maximum_width"] = (
+            k_factor
+            * (
+                200.0
+                * 1.1
+                * (
+                    (mtow * sizing_factor_ultimate / (10.0**5.0)) ** 0.286
+                    * (fus_length * 3.28084 / 10.0) ** 0.857
+                    * 3.28084
+                    / 10.0
+                    * (v_max_sl / 100.0) ** 0.338
+                )
+                ** 1.1
+                * (maximum_width + maximum_height) ** 0.1
+            )
+        )
```

Comment:
For some reasons, GitHub actions haven't triggered. Which means there was no automated check from ruff and it seems like it needs one. Could you check that ruff won' reformat anything ?